web/src/main/webapp/table.ts

```diff
@@ -772,12 +772,38 @@ export class TableView extends RemoteObject
             return "?";  // TODO
     }
 
+    public drawDataRange(canvas: HTMLCanvasElement, position: number, count: number) : void {
+        var ctx = canvas.getContext('2d');
+
+        // A little hack to make canvas look prettier on retina displays.
```

Comment:
So far we don't have any target-specific code.
Ideally this would be done with CSS instead of straight code.